people/MizuharaNagisa/page.en.md

```diff
@@ -12,11 +12,11 @@ She was friends with [Ying](https://www.one-among.us/profile/Uekawakuyuurei/).
 During her life, she was an active long-term contributor for Chinese Wikipedia and other MediaWiki sites.
 Since 2019, she started playing *Kancolle*.
 Her favourite character was Destroyer Hibiki.
-Her hobbies were tinkering with computer hardware like Respberry Pi and drawing.
+Her hobbies were tinkering with computer hardware like Raspberry Pi and drawing.
 She was also proficient in Japanese.
 
 Because of depression and other problems, she had to stop going to school since 2019;
-because of inacceptance of her family and other problems she faced, her depression and anxiety got worse and started to suffer from insomnia.
+because of lack of understanding from her family and other problems she faced, her depression and anxiety got worse and started to suffer from insomnia.
```

Comment:
Should've been unacceptance.

Reply:
Thank you, I thought of that too but wasn't too sure >.<